pkg/utils/nodepool/nodepool.go

```diff
@@ -40,15 +40,3 @@ func List(ctx context.Context, c client.Client, opts ...client.ListOption) (*v1b
 	}
 	return nodePoolList, nil
 }
-
```

Comment:
Ideally, I think we can drop this file entirely if we are removing the utils here